apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/functions/OnboardingFunctions.java

```diff
@@ -36,14 +37,20 @@ public class OnboardingFunctions {
 
     private final OnboardingService service;
     private final CompletionService completionService;
+    private final MessageService messageService;
 
     private final ObjectMapper objectMapper;
     private final TaskOptions optionsRetry;
 
-    public OnboardingFunctions(OnboardingService service, ObjectMapper objectMapper, RetryPolicyConfig retryPolicyConfig, CompletionService completionService) {
+    public OnboardingFunctions(OnboardingService service,
```

Comment:
you could create another class for this specific use case, for ex. NotificationFunctions

Reply:
As you suggested, I created a separated class to host logic for notifications functions